docs/reference/troubleshooting/discovery-issues.asciidoc

```diff
@@ -118,3 +119,79 @@ include::network-timeouts.asciidoc[tag=troubleshooting-network-timeouts-threads]
 If a node joins the cluster but {es} determines it to be faulty then it will be
 removed from the cluster again. See <<cluster-fault-detection-troubleshooting>>
 for more information.
+
+[discrete]
+[[discovery-master-split-brain]]
+=== Multiple masters are elected
+
+Infrequently, {es} can enter a {wikipedia}/Split-brain_(computing)[split-brain] 
+state, where nodes within a cluster disagree on which node is elected 
+as master node. This is a non-viable state for the cluster, and can lead to 
```

Comment:
This is not true, since 7.0 there is no path to enter such a state. See https://www.elastic.co/blog/a-new-era-for-cluster-coordination-in-elasticsearch

Reply:
1. Kindly port over to [cloud#128568](https://github.com/elastic/cloud/issues/128568) for a sub-discussion which discusses code paths claiming multiple ES masters on 8.x but requires sharing company private data. 
2. "split brain" is still referenced across ES docs, e.g. [from Google](https://www.google.com/search?client=firefox-b-1-d&q=%28%22split-brain%22+or+%22split+brain%22%29+site%3A%2F%2Fhttps%3A%2F%2Fwww.elastic.co%2Fguide%2Fen%2Felasticsearch%2Freference%2Fcurrent). Could it potentially be that my definition is surprising to you but split-brain is still possible as still referenced?
    <img width="667" alt="image" src="https://github.com/elastic/elasticsearch/assets/26751266/3114ba2c-3864-4782-a6b8-72b547daa9d9">

Reply:
All of the links in this screenshot are to documentation describing how ES does _not_ suffer from split-brain.

Reply:
The other review points are details on which we can iterate but I think this is a fundamental problem with the PR as it currently stands. ES cannot in fact enter the split-brain/multiple-master state described here, so it doesn't make sense to say that it can nor to describe how to remedy the situation. I don't really want to just close the PR because I'm guessing there's a good reason you want to add some of this information to the docs, but I can't really see what that is right now. Could you rework this bit to be more accurate?